tpp-app/App.js

```diff
@@ -48,13 +47,7 @@ const CustomTabBarButton = ({ onPress }) => {
         alignItems: 'center',
         justifyContent: 'center'
       }}>
-        <Image
-          style={{
-            width: 24.67,
-            height: 30.83
-          }}
-          source={icon}
-        />
+        {icon}
```

Comment:
Accompanies the other comment
```suggestion
        <VectorImage source={require(icon)} />
```